src/templates/community-champion-profile.js

```diff
@@ -309,15 +267,29 @@ const CommunityChampionProfile = props => {
                             }
                         />
                     </ImageContainer>
-                    <QuoteSection quote={quote} />
+                    {quote && <QuoteSection quote={quote} />}
                     <TextContainer>
                         <Name>{fullName}</Name>
-                        <InfoSection
+                        <BioHeader
                             occupation={title}
                             location={location}
                             languages={languagesSpoken}
                         />
                         <Bio>{bio}</Bio>
+                        {(BlogsAndPublications || Socials) && (
```

Comment:
Similar thought, could we have `Socials` be empty but exist similar to how we have with `Certifications`? What about `BlogsAndPublications`?

Reply:
`Socials` yes, `BlogsAndPublications` no. this is because I made `BlogsAndPublications` a repeatable component where the fields in each one are required